development/ts-migration-dashboard/scripts/write-list-of-files-to-convert.ts

```diff
@@ -30,30 +35,42 @@ async function main(): Promise<void> {
         );
       }),
     )
-  ).flat();
+  )
+    .flat()
```

Comment:
Here we're making sure that we don't give `madge` files it doesn't need (we do a similar thing in `build-module-partitions.ts`).